vcxagency-node/Dockerfile

```diff
@@ -58,6 +58,8 @@ ENV PG_WALLET_MIN_IDLE_COUNT ${PG_WALLET_MIN_IDLE_COUNT:-1}
 ENV PG_WALLET_MAX_CONNECTIONS ${PG_WALLET_MAX_CONNECTIONS:-'90'}
 ENV PG_WALLET_CONNECTION_TIMEOUT_MINS ${PG_WALLET_CONNECTION_TIMEOUT_MINS:-'30'}
 
+ENV MAX_REQUESTS_PER_MINUTE ${MAX_REQUESTS_PER_MINUTE:-'30'}
```

Comment:
I'd go by default for esentially no-limit, like 100000 or something :-) I think should be very well aware what the rate limit is (and if there's even any)